.ci_support/migrations/pytorch25.yaml

```diff
@@ -0,0 +1,10 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for pytorch 2.5
+  kind: version
+  migration_number: 1
+libtorch:
+- '2.5'
+migrator_ts: 1730666768.682698
+pytorch:
+- '2.5'
```

Comment:
Noticed we still have a migrator for PyTorch 2.5. Do we still need this? Or can it be dropped?

Reply:
Just look at the provenance here - these files come from old migrators and will be dropped upon rerendering.

Reply:
That was certainly my understanding when looking things over

Weirdly re-rendering locally did not drop the files for me. So not sure why that is

Can manually remove them

Reply:
Looks like the bot cleared them: https://github.com/conda-forge/dgl-feedstock/pull/57/commits/c8c8800d44e7b847508ca9ba38dea545096fb93d

Not sure why it didn't work locally